src/components/ApplicationForm/Skills.js

```diff
@@ -83,37 +83,71 @@ export default ({ errors, formInputs, onChange, role, handleResume }) => {
           </FormRow>
 
           {role === 'designer' ? (
-            <FormRow id="portfolio">
-              <TextInput
-                inline
-                placeholder="Optional"
-                size="large"
-                value={formInputs.portfolio}
-                invalid={!!errors.portfolio}
-                errorMsg={errors.portfolio}
-                onChange={e =>
-                  onChange({
-                    portfolio: e.target.value,
-                  })
-                }
-              />
-            </FormRow>
+            <>
+              <FormRow id="Personal website/portfolio link">
+                <TextInput
+                  inline
+                  placeholder="Required"
+                  size="large"
+                  value={formInputs.portfolio}
+                  invalid={!!errors.portfolio}
+                  errorMsg={errors.portfolio}
+                  onChange={e =>
+                    onChange({
+                      portfolio: e.target.value,
+                    })
+                  }
+                />
+              </FormRow>
+              <FormRow id="GitHub/BitBucket/GitLab">
```

Comment:
Don't think we can use `/` in `id` either

Reply:
see above